src/main/java/dev/ebrydeu/spring_boot_library/domain/entities/User.java

```diff
@@ -0,0 +1,24 @@
+package dev.ebrydeu.spring_boot_library.domain.entities;
+
+import lombok.*;
+import jakarta.persistence.*;
+import java.util.UUID;
+@Data
+@Entity
+@Table(name = "USER")
```

Comment:
Use [lowercase](https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html) instead.
```suggestion
@Table (name = "user")
```